frontend/src/modules/2DViewer/settings/settings.tsx

```diff
@@ -19,7 +20,7 @@ import { selectedFieldIdentifierAtom } from "./atoms/derivedAtoms";
 import { DataProviderManagerWrapper } from "./components/dataProviderManagerWrapper";
 
 export function Settings(props: ModuleSettingsProps<any>): React.ReactNode {
-    const ensembleSet = useEnsembleSet(props.workbenchSession);
+    const ensembleSet = usePublishSubscribeTopicValue(props.workbenchSession, WorkbenchSessionTopic.EnsembleSet);
```

Comment:
This is a good example where I'd like to still see an `useEnsembleSet`-hook if possible.

If fixed, replace `usePublishSubscribeTopicValue(props.workbenchSession, WorkbenchSessionTopic.EnsembleSet);` with `useEnsembleSet` all other places in the code base as well.